gophish-tools/lipca-setup.sh

```diff
@@ -0,0 +1,335 @@
+  sudo docker run -it --rm --workdir="$CISA_HOME" -v "$TEMPLATE_PATH":"$CISA_HOME":Z "$TOOLS_IMAGE_NAME" "$TEMPLATE_ALIAS" -t && sudo vi "$TEMPLATE_TARGETS_FILENAME"
+}
+
+create_email_template() {
+  # Runs pca-wizard-templates tool in gophish-tools container and outputs
+  # a pre-formatted json file named "template_email.json" file in the specified
+  # directory and open vim to edit the template as needed for modification.
+  sudo docker run -it --rm --workdir="$CISA_HOME" -v "$TEMPLATE_PATH":"$CISA_HOME":Z "$TOOLS_IMAGE_NAME" "$TEMPLATE_ALIAS" -e && sudo vi "$TEMPLATE_EMAIL_FILENAME"
+}
+
+email_template_prompt() {
+  # Prompt user and ask if template generation is needed.
+  while true; do
+    read -rp "Do you need an email template file generated? (yes/no) " yn
+    case $yn in
+      [Yy]*)
+        create_email_template
+        break
+        ;;
+      [Nn]*) echo "Skipping email template creation and proceeding with setup." && break ;;
+      *) echo "Please answer yes or no." ;;
+    esac
+  done
+}
+
+target_template_prompt() {
+  # Prompt user and ask if template generation is needed.
+  while true; do
+    # Input R
+    read -rp "Do you need an targets template file generated? (yes/no) " yn
+    case $yn in
+      [Yy]*)
+        create_target_template
+        break
+        ;;
+      [Nn]*)
+        echo "Skipping targets template creation and proceeding with setup."
+        break
+        ;;
+      *) echo "Please answer yes or no." ;;
+    esac
+  done
+}
+
+#=============================
+#     ASSESSMENT TOOLS
+#=============================
+
+create_assessment() {
+  # Runs the pca-wizard tool to setup a new campaign generate and saves the
+  # output json to the lipca-temp directory for importing
+  read -rp 'Enter assessment id/name for new assessment: ' id
+  read -rp 'Enter level of the new assessment (1-6): ' level
+
+  # TODO: Determine if we want to keep mix of underscores and hypens in level naming.
+  ASSESSMENT_NAME="$id"_level-"$level"
+  FULL_ASSESSMENT_PATH="$ASSESSMENT_PATH/$ASSESSMENT_NAME"
+
+  # Copy templates for injestion
+  echo "Copying template files named template_* for use in setup process."
+  sudo cp /share/PCA/templates/template_* "$TEMPLATE_INJESTION_PATH"
+  echo "Templates copied to: $TEMPLATE_INJESTION_PATH"
+
+  # Run using docker gophish-tools image pca-wizard
+  sudo docker run -it --rm --workdir="$CISA_HOME" -v "$ASSESSMENT_PATH":"$CISA_HOME":Z "$TOOLS_IMAGE_NAME" "$WIZARD_ALIAS" "$ASSESSMENT_NAME"
+  echo "Saved Assessment: $FULL_ASSESSMENT_PATH"
+}
+
+import_assessment() {
+  # Runs the import-assessment.sh script to import the generated assessment.json
+  # data in the lipca-temp dir (generated from create_assessment)
+  "$IMPORT_ASSESSMENT_PATH" "$FULL_ASSESSMENT_PATH".json
+}
+
+export_assessment() {
+  # Runs the export-assessment.sh script to export the generated assessment
+  # data in the lipca-temp dir (generated from create_assessment)
```

Comment:
I think this is what is meant here, but please let me know if I'm wrong:
```suggestion
  # data to the lipca-temp dir (generated from create_assessment)
```